modules/rubiconBidAdapter.js

```diff
@@ -136,17 +138,124 @@ var sizeMap = {
   578: '980x552',
   580: '505x656'
 };
+
 _each(sizeMap, (item, key) => sizeMap[item] = key);
 
+export const converter = ortbConverter({
+  request(buildRequest, imps, bidderRequest, context) {
+    const {bidRequests} = context;
+    const data = buildRequest(imps, bidderRequest, context);
+    data.test = config.getConfig('debug') ? 1 : 0;
+    deepSetValue(data, 'ext.prebid.cache', {
+      vastxml: {
+        returnCreative: rubiConf.returnVast === true
+      }
+    });
+
+    deepSetValue(data, 'ext.prebid.bidders', {
+      rubicon: {
+        integration: rubiConf.int_type || DEFAULT_PBS_INTEGRATION,
+      }
+    });
+
+    deepSetValue(data, 'ext.prebid.targeting.pricegranularity', getPriceGranularity(config));
+
+    let modules = (getGlobal()).installedModules;
+    if (modules && (!modules.length || modules.indexOf('rubiconAnalyticsAdapter') !== -1)) {
+      deepSetValue(data, 'ext.prebid.analytics', {'rubicon': {'client-analytics': true}});
+    }
+
+    addOrtbFirstPartyData(data, bidRequests);
+
+    delete data?.ext?.prebid?.storedrequest;
+
+    // floors
+    if (rubiConf.disableFloors === true) {
+      delete data.ext.prebid.floors;
+    }
+
+    // If the price floors module is active, then we need to signal to PBS! If floorData obj is present is best way to check
+    const haveFloorDataBidRequests = bidRequests.filter(bidRequest => typeof bidRequest.floorData === 'object');
+    if (haveFloorDataBidRequests.length > 0) {
+      data.ext.prebid.floors = { enabled: false };
+    }
+    // multibid
+    const multibid = config.getConfig('multibid');
+    addMultibid(data, multibid);
+
+    const aliasedBidRequests = bidRequests
```

Comment:
Won't the `setRequestExtPrebidAliases` function handle this? 

I thought the ORTB converter thing handled most of the usual basic stuff?

Reply:
right, removing code to set alias

Reply:
👍